apps/hashdotdev/src/pages/roadmap.page.tsx

```diff
@@ -0,0 +1,268 @@
+                </Box>
+                {label}
+              </Typography>
+            </Link>
+          ))}
+        </Box>
+      </Box>
+    </Box>
+  </Container>
+);
+
+const UseCases: FunctionComponent = () => (
+  <Container sx={{ marginBottom: 10 }}>
+    <AnchorTag id="use-cases" />
+    <Typography variant="hashHeading3" gutterBottom>
+      Use Cases
+    </Typography>
+    <Box
+      display="flex"
+      gap={2}
+      flexWrap="wrap"
+      sx={{ justifyContent: { xs: "space-between", sm: "flex-start" } }}
+    >
+      {useCases.map(({ name, icon }) => (
+        /** @todo: make clickable when docs pages exist for each item */
+        // <Link key={name} href={href}>
+        <Box
+          key={name}
+          sx={{
+            display: "flex",
+            flexDirection: "column",
+            width: 135,
+            padding: ({ spacing }) => spacing(2, 1.5),
+            borderRadius: "8px",
+            borderStyle: "solid",
+            borderWidth: 1,
+            borderColor: ({ palette }) => palette.gray[30],
+          }}
+        >
+          {icon}
+          <Typography
+            sx={{
+              marginTop: 1.25,
+              textAlign: "center",
+              fontSize: 15,
+              fontWeight: 500,
+              color: ({ palette }) => palette.gray[80],
+              lineHeight: 1.2,
+            }}
+          >
+            {name}
+          </Typography>
+        </Box>
+        // </Link>
+      ))}
+    </Box>
+  </Container>
+);
+
+const GetInvolved: FunctionComponent = () => (
+  <Container sx={{ marginBottom: 10 }}>
+    <AnchorTag id="get-involved" />
+    <Typography variant="hashHeading3" marginBottom={5}>
+      Get Involved
+    </Typography>
+    <Typography variant="hashHeading4" marginBottom={3}>
+      <FaIcon
+        name="wind"
+        type="light"
+        sx={{ color: ({ palette }) => palette.teal[90], marginRight: 3 }}
+      />
+      Become an early adopter
+    </Typography>
+    <Typography marginBottom={1}>
+      <strong>Interested in using HASH?</strong>
+    </Typography>
+    <Box
+      component="ul"
+      sx={{
+        marginTop: 0,
+        marginBottom: 2,
+        listStyle: "none",
+        paddingLeft: 0,
+        "> li": {
+          marginBottom: 0.5,
+          svg: {
+            position: "relative",
+            top: 7,
+            marginRight: 2,
+            fontSize: 16,
+          },
+        },
+      }}
+    >
+      <Box component="li" display="flex">
+        <FaIcon name="arrow-right" type="regular" />
+        <Typography>
+          <Link href="https://app.hash.ai/signup" openInNew>
+            <strong>Create an account</strong>
+          </Link>{" "}
+          to try out the hosted version of HASH
+        </Typography>
+      </Box>
+      <Box component="li" display="flex">
+        <FaIcon name="arrow-right" type="regular" />
+        <Typography>
+          {/* @todo: link to https://github.com/hashintel/hash */}
```

Comment:
Anything stopping us from doing this now?

Reply:
This isn't meant to link to `github.com/hashintel/hash` but the hash.dev/docs _getting started_ page.

It can link to the GitHub page as a stop-gap, though. Let's just update the link on the open branch that adds the `hash.dev/docs` section.

Reply:
Added the link in https://github.com/hashintel/hash/pull/2747/commits/bd8d6b789cb9beb95db5645b5a412ff9f5e28d7b